src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ObjectNodeSection.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace ILCompiler.DependencyAnalysis
+{
+    [Flags]
+    public enum CustomSectionAttributes
```

Comment:
Yeah I initially had another enum and removed it because it looked obtuse to convert between two identical enums :(  I can add it back in.